modules/reindex/src/test/resources/rest-api-spec/test/reindex/10_basic.yaml

```diff
@@ -0,0 +1,366 @@
+---
+"Response format for created":
```

Comment:
From here on out it is just miles of REST tests, exactly copied from master. The only thing missing is validation that you don't use "from".